[2022-11-02T11:50:01.434+0700] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: my_python_op_dag.third_task manual__2022-11-02T04:49:57.173105+00:00 [queued]>
[2022-11-02T11:50:01.439+0700] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: my_python_op_dag.third_task manual__2022-11-02T04:49:57.173105+00:00 [queued]>
[2022-11-02T11:50:01.439+0700] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-02T11:50:01.439+0700] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2022-11-02T11:50:01.439+0700] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-02T11:50:01.460+0700] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): third_task> on 2022-11-02 04:49:57.173105+00:00
[2022-11-02T11:50:01.461+0700] {standard_task_runner.py:55} INFO - Started process 41338 to run task
[2022-11-02T11:50:01.464+0700] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'my_python_op_dag', 'third_task', 'manual__2022-11-02T04:49:57.173105+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/python_op_dag.py', '--cfg-path', '/tmp/tmp430w_mdl']
[2022-11-02T11:50:01.464+0700] {standard_task_runner.py:83} INFO - Job 28: Subtask third_task
[2022-11-02T11:50:01.503+0700] {task_command.py:376} INFO - Running <TaskInstance: my_python_op_dag.third_task manual__2022-11-02T04:49:57.173105+00:00 [running]> on host ginraca-VirtualBox
[2022-11-02T11:50:01.547+0700] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=my_python_op_dag
AIRFLOW_CTX_TASK_ID=third_task
AIRFLOW_CTX_EXECUTION_DATE=2022-11-02T04:49:57.173105+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-02T04:49:57.173105+00:00
[2022-11-02T11:50:01.552+0700] {logging_mixin.py:120} INFO - Hello World! My name is None and it is currently None where i'm at
[2022-11-02T11:50:01.552+0700] {python.py:177} INFO - Done. Returned value was: None
[2022-11-02T11:50:01.555+0700] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=my_python_op_dag, task_id=third_task, execution_date=20221102T044957, start_date=20221102T045001, end_date=20221102T045001
[2022-11-02T11:50:01.605+0700] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-02T11:50:01.613+0700] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
